pkg/kn/commands/service/service.go

```diff
@@ -66,8 +66,13 @@ func showUrl(client serving_kn_v1alpha1.KnServingClient, serviceName string, ori
 	revisionUpdateStatus := ""
 	newRevision := service.Status.LatestReadyRevisionName
 	if originalRevision != "" && originalRevision == newRevision {
-		revisionUpdateStatus = " (unchanged)"
+		revisionUpdateStatus = "unchanged"
```

Comment:
Instead of introducing this temporary variable to check later on, I suggest to directly print the two different messages directly in the two branches of this if statement and remove the second if.

Reply:
OK that's fair. See update. Pushing now. Thanks.